src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -26,11 +27,12 @@ use kata_types::{
 };
 use nix::mount::MsFlags;
 use persist::sandbox_persist::Persist;
+use std::cmp::Ordering;
 use std::{collections::HashSet, fs::create_dir_all};
 
 const DRAGONBALL_KERNEL: &str = "vmlinux";
 const DRAGONBALL_ROOT_FS: &str = "rootfs";
-
+const BALLOON0: &str = "balloon0";
```

Comment:
what about BALLOON_DEVICE_ID? and why mem device don't use a const name?

Reply:
fixed